src/um/dbghelp.rs

```diff
@@ -385,3 +385,121 @@ STRUCT!{struct IMAGEHLP_LINEW64 {
     Address: DWORD64,
 }}
 pub type PIMAGEHLP_LINEW64 = *mut IMAGEHLP_LINEW64;
+
+extern "system" {
+    pub fn EnumDirTree(
+        hProcess: HANDLE, RootPath: PCSTR, InputPathName: PCSTR, OutputPathBuffer: PSTR,
```

Comment:
All function arguments should be on a line by themselves. See the style guidelines.